### ISSUE TITLE
Json schema: Select best match in oneOf while loading form

### ISSUE THREAD

**Issue by rishabh413** [ngx-formly/ngx-formly#3714]
**Is your feature request related to a problem? Please describe.**

I have this schema:
```json
{
  "type": "object",
  "properties": {
    "heading": {
      "title" : "Heading",
      "oneOf" : [
       { 
        
        "title" : "Option 1: Standard heading",
        "type": "string",
        "oneOf" : [
          {  "const" : "Heading 1" }, { "const" : "Heading 2" }, { "const" : "Heading 3" }
        ] 
       },
       {        
        "title" : "Option 2: Custom heading",      
        "type": "string"
        }
      ]
    }
  }
}
```

and its rendered correctly. The user selects  "Option 1: Standard heading" and then selects "Heading 2" and below data is persisted in json.
```json
{
  "heading": "Heading 2"
}
```
But, when we open the form again with persisted data, we see the selection to be "Option 2: Custom heading" and "Heading 2" filled as string input in it.

**Describe the solution you'd like**

We understand that both the alternates in oneOf match the persisted json. Can we have "most strictly" matching oneOf decided by ngx-formly and hence when form loads, it shows "Option 1: Standard heading". OR maybe there is some way to persist what was selected by user in the oneOf selection.

**Describe alternatives you've considered**

None

**Additional context**

None

**Comment by rishabh413** [comment]
Duplicate of #3723 3723

**Comment by rishabh413** [comment]
Duplicate of #3723

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.2.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.2.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
